test/resque_test.rb

```diff
@@ -219,17 +219,17 @@
     end
 
     it "can peek at a queue" do
-      assert_equal({ 'name' => 'chris' }, Resque.peek(:people))
+      assert_equal({ 'name' => 'mark' }, Resque.peek(:people))
```

Comment:
Now the behavior of `peek` has changed. While before we would get the next job that would be executed now we would get the last one.